android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -135,10 +136,12 @@ public class RNSentryModuleImpl {
     /** Max trace file size in bytes. */
     private long maxTraceFileSize = 5 * 1024 * 1024;
 
+  private final RNSentryTimeToDisplay timeToDisplay;
```

Comment:
`l`: Minor formating nitpick
```suggestion
     private final RNSentryTimeToDisplay timeToDisplay;
     
```